src/NosCore.Parser/Parsers/ItemParser.cs

```diff
@@ -0,0 +1,643 @@
+        {
+            return Convert.ToByte(ImportItemType(chunk) switch
+            {
+                ItemType.Armor => chunk["DATA"][0][10],
+                ItemType.Weapon => chunk["DATA"][0][10],
+                _ => "0"
+            });
+        }
+        private ItemEffectType ImportEffect(Dictionary<string, string[][]> chunk)
+        {
+            return (ItemEffectType)Convert.ToUInt16(ImportItemType(chunk) switch
+            {
+                ItemType.Special => chunk["DATA"][0][2],
+                ItemType.Box => chunk["DATA"][0][2],
+                ItemType.Magical => chunk["DATA"][0][2],
+                ItemType.Production => chunk["DATA"][0][2],
+                ItemType.Map => chunk["DATA"][0][2],
+                ItemType.Main => chunk["DATA"][0][2],
+                ItemType.Teacher => chunk["DATA"][0][2],
+                ItemType.Upgrade => chunk["DATA"][0][2],
+                _ => "0"
+            });
+        }
+
+        private EquipmentType ImportEquipmentType(Dictionary<string, string[][]> chunks) => (EquipmentType)Enum.Parse(typeof(EquipmentType),
+                chunks["INDEX"][0][5] != "-1" ? chunks["INDEX"][0][5] : "0");
+
+        private ItemType ImportItemType(Dictionary<string, string[][]> chunk) => chunk["INDEX"][0][3] != "-1"
+                    ? (ItemType)Enum.Parse(typeof(ItemType), $"{(short)ImportType(chunk)}{chunk["INDEX"][0][3]}")
+                    : ItemType.Weapon;
+
+        private NoscorePocketType ImportType(Dictionary<string, string[][]> chunks)
+        {
+            return Convert.ToByte(chunks["INDEX"][0][2]) switch
+            {
+                4 => NoscorePocketType.Equipment,
+                8 => NoscorePocketType.Equipment,
+                9 => NoscorePocketType.Main,
+                10 => NoscorePocketType.Etc,
+                _ => (NoscorePocketType)Enum.Parse(typeof(NoscorePocketType), chunks["INDEX"][0][2])
+            };
+        }
+
+        private byte ImportResistance(Dictionary<string, string[][]> chunk, ElementType element)
+        {
+            var equipmentType = ImportEquipmentType(chunk);
+            return Convert.ToByte(element switch
+            {
+                ElementType.Fire => equipmentType == EquipmentType.Sp ? chunk["DATA"][0][15] : chunk["DATA"][0][7],
+                ElementType.Light => equipmentType == EquipmentType.Sp ? chunk["DATA"][0][17] : chunk["DATA"][0][9],
+                ElementType.Water => equipmentType == EquipmentType.Sp ? chunk["DATA"][0][16] : chunk["DATA"][0][8],
+                ElementType.Dark => equipmentType == EquipmentType.Sp ? chunk["DATA"][0][18] : chunk["DATA"][0][11],
+                _ => "0"
+            });
+        }
+
+        private void SetVehicles(List<ItemDto> items, Dictionary<byte, List<(short, short)>> vehicleDictionary)
+        {
+            foreach (var vehicle in vehicleDictionary)
+            {
+                var speed = vehicle.Key;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'speed'.](https://app.codacy.com/app/NosCoreIO/NosCore/pullRequest?prid=4456396)